src/lib/context-menu.ts

```diff
@@ -255,6 +256,36 @@ export class ContextMenu {
         })
       )
     }
+    if (type === schema.nodes.table_element) {
+      let addGeneralNote = true
+      const tableElementFooter = findChildrenByType(
+        this.node,
+        schema.nodes.table_element_footer
+      )
+      if (tableElementFooter.length) {
+        const paragraphs = findChildrenByType(
+          tableElementFooter[0].node,
+          schema.nodes.paragraph,
+          false
+        )
+        if (paragraphs.length) {
+          addGeneralNote = false
+        }
+      }
+      if (addGeneralNote) {
+        menu.appendChild(
+          this.createMenuSection((section: HTMLElement) => {
+            section.appendChild(
+              this.createMenuItem('Add General Note', () => {
+                const { state, dispatch } = this.view
+                insertGeneralFootnote(this.node, this.getPos(), state, dispatch)
+                popper.destroy()
```

Comment:
Do you need this? I was thinking it would work without it already, as far as I remember we have some listeners.

Reply:
done